godot/scenes/combat_arena/combat_arena.gd

```diff
@@ -81,8 +86,7 @@ func _on_end_turn_button_pressed():
 		alive_enemies.assign(get_alive_enemies().map(func (node): return node.get_enemy_node()))
 		action.process_action(player_node, alive_enemies , action_entry.params, enemy)
 		enemy.update_intent()
-		ingredient_pouch.clear_hand()
-		ingredient_pouch.draw_hand()
```

Comment:
With the 1s delay, these two lines caused the player hand to glitch and look weird, so I moved these up.